app/Data/migrations/versions/387d4c07cc85_picture_table.py

```diff
@@ -10,15 +10,29 @@
 
 
 # revision identifiers, used by Alembic.
+from sqlalchemy.engine.reflection import Inspector
+
 revision = '387d4c07cc85'
 down_revision = 'a152efd05762'
 branch_labels = None
 depends_on = None
 
 
 def upgrade():
-    pass
+    op.create_table(
```

Comment:
``sa.Column('UserId', sa.Integer, sa.ForeignKey('review.Id'), nullable=False)``
Should point to ``sa.ForeignKey('user.Id')`` instead of ``sa.ForeignKey('review.Id')``

 sa.Column('LocationId', sa.Integer, sa.ForeignKey('user.Id'), nullable=False)
Should point to ``sa.ForeignKey('location.Id)`` instead of ``sa.Integer, sa.ForeignKey('user.Id')``